ParisBNF/abb/BNFabb87.xml

```diff
@@ -0,0 +1,63 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb87" xml:lang="en" type="mss">
```

Comment:
You can leave it to Jonas if you wish, sure, but this is an item with no images available online, so a special competence for chant collections will be needed only for item 1, and to a restricted degree. You can still do the description nearly entirely (assign ids to works, additional note, what is available from codicological features), Jonas would later elaborate at a convinient. It is all the same, just less gaps.

Reply:
Ok